etcdctl/ctlv3/command/ep_command.go

```diff
@@ -83,9 +83,10 @@ func epHealthCommandFunc(cmd *cobra.Command, args []string) {
 	sec := secureCfgFromCmd(cmd)
 	dt := dialTimeoutFromCmd(cmd)
 	auth := authCfgFromCmd(cmd)
+	discovery := discoveryCfgFromCmd(cmd)
 	cfgs := []*v3.Config{}
 	for _, ep := range endpointsFromCluster(cmd) {
-		cfg, err := newClientCfg([]string{ep}, dt, sec, auth)
+		cfg, err := newClientCfg([]string{ep}, dt, sec, auth, discovery)
```

Comment:
the endpoints were already derived from the cluster,  there's nothing for discovery to do here; leave unchanged

Reply:
removed